style.css

```
@@ -0,0 +1,133 @@
+:root {
+    --font-family-heading: "Anton", sans-serif;
+    --font-family-body: "Noto Sans JP", sans-serif;
+
+    --color-white: #fffefc;
+    --color-black: #020911;
+    --color-accent-red: #f20574;
+    --color-accent-yellow: #f2e52e;
+    --color-background-black: #171525;
+    --color-background-navy: #030427;
+    --color-form-background-yellow: #fffab9;
+    --color-form-placeholder-gray: #ad9783;
+    --color-form-border-red: #df6da2;
+    --color-button-hover-red: #f043a5;
+    --color-course-hover-yellow: #f4e877;
+    --color-hero-overlay: rgba(57, 24, 111, 0.4);
+    --color-course-overlay: rgba(32, 8, 19, 0.63);
+}
+
+/* global */
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: var(--font-family-body);
+}
+
+img {
+    display: block;
+}
+
+/* header */
+.header {
+    height: 80px;
+    background-color: var(--color-black);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* ヘッダーの追従 */
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    width: 100%;
+}
+
+.header-logo-wrapper {
+    margin: 0;
+}
+
```

```suggestion
    .header-logo {
        height: 50px;
    }
```
画面幅750px以上の際にロゴの大きさが変化しないため高さを調整